components/git/metadata.js

```diff
@@ -83,15 +83,19 @@ function handler(argv) {
   const logStream = process.stdout.isTTY ? process.stdout : process.stderr;
   const cli = new CLI(logStream);
 
-  return getMetadata(Object.assign(
-    {}, config, argv, parsed
-  ), cli).catch((err) => {
-    if (cli.spinner.enabled) {
-      cli.spinner.fail();
-    }
-    cli.error(err);
-    process.exit(-1);
-  });
+  return getMetadata(Object.assign({}, config, argv, parsed), cli)
+    .then(({status}) => {
+      if (status === false) {
+        process.exit(1);
```

Comment:
I think we can just let this throw here so it goes to the catch below, and replace `process.exit(-1)` below to `process.exit(-1)`? (I don't even know what I was thinking when I put `-1` there :/)

Reply:
I think you mean `process.exit(1)`?

Reply:
Yes, like so

```
    .then(({status}) => {
      if (status === false) {
        throw new Error(`PR checks failed`);
      }
    })
    .catch((err) => {
      if (cli.spinner.enabled) {
        cli.spinner.fail();
      }
      cli.error(err);
      process.exit(1);
    });

```

Reply:
Cool. That was exactly what I had been thinking about.